<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClockWidget</class>
 <widget class="QWidget" name="ClockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>175</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Widget {
border: none;
border-top-left-radius: 4px;
border-top-right-radius: 4px;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0.01 rgba(120, 120, 120),
stop: 0.05 rgba(150, 150, 150),
stop: 0.5 rgba(120, 120, 120),
stop: 0.52 rgba(80, 80, 80),
stop: 0.9 rgba(50, 50, 50),
stop: 1.0 rgba(50, 50, 50));
}

#Widget { 
padding: 3px;
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0.0 rgba(95, 95, 95),
stop: 0.15 rgba(75, 75, 75),
stop: 0.9 rgba(95, 95, 95),
stop: 1.0 rgba(45, 45, 45));
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-left-radius: 4px;
border-bottom-right-radius: 4px;
}

QPushButton {
padding: 4px;
border-radius: 4px;
border: 1px solid rgba(55, 55, 55);
font-weight: bold;
/*
background: qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(25, 25, 25),
stop: 0.05 rgba(75, 75, 75),
stop: 0.15 rgba(125, 125, 125),
stop: 0.85 rgba(75, 75, 75),
stop: 1.0 rgba(25, 25, 25));
*/
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0.01 rgba(120, 120, 120),
stop: 0.05 rgba(150, 150, 150),
stop: 0.5 rgba(120, 120, 120),
stop: 0.52 rgba(80, 80, 80),
stop: 0.9 rgba(50, 50, 50),
stop: 1.0 rgba(50, 50, 50));
color: rgba(225, 225, 225);
}

QPushButton:pressed {
background: qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(25, 25, 25),
stop: 0.05 rgba(25, 25, 25),
stop: 0.15 rgba(75, 75, 75),
stop: 0.85 rgba(25, 25, 25),
stop: 1.0 rgba(0, 0, 0));
color: rgba(200, 200, 200);
}

QPushButton:disabled {
background:qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(50, 50, 50),
stop: 0.05 rgba(100, 100, 100),
stop: 0.15 rgba(150, 150, 150),
stop: 0.85 rgba(100, 100, 100),
stop: 1.0 rgba(50, 50, 50));
}

QLabel {
color: rgba(35, 35, 35);
font-weight: normal;
background: transparent;
border: none;
}

QCheckBox {
background: transparent;
border: none;
}
/**
QPushButton {
border-radius: 4px;
border: 1px solid rgba(75, 75, 75);
background: qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(25, 25, 25),
stop: 0.05 rgba(75, 75, 75),
stop: 0.15 rgba(125, 125, 125),
stop: 0.85 rgba(75, 75, 75),
stop: 1.0 rgba(25, 25, 25));
color: rgba(225, 225, 225);
}
/**
QPushButton:pressed {
background:qlineargradient(x1:0, y1:0, x2: 0, y2:1,
stop: 0.0 rgba(0, 0, 0),
stop: 0.05 rgba(50, 50, 50),
stop: 0.15 rgba(100, 100, 100),
stop: 0.85 rgba(50, 50, 50),
stop: 1.0 rgba(0, 0, 0));
color: rgba(200, 200, 200);
}**/

#controlsFrame {
	background: rgba(200, 200, 200);
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-left-radius: 2px;
border-bottom-right-radius: 2px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
stop: 0.0 rgba(125, 125, 125),
stop: 0.02 rgba(200, 200, 200),
stop: 0.98 rgba(200, 200, 200),
stop: 1.0 rgba(125, 125, 125));
}

QLineEdit {
border: 1px solid rgba(175, 175, 175);
background: rgba(255, 255, 255);
border-top-left-radius: 4px;
border-top-right-radius: 4px;
border-bottom-right-radius: 4px;
border-bottom-left-radius: 4px;
}

QGroupBox {
border: 1px solid rgba(175, 175, 175);
border-radius: 4px;
margin-top: 1ex;
}

QGridLayout {
border: 1px solid rgba(175, 175, 175);
border-radius: 4px;
margin: 10px;
}

#simulatorButton {
width: 100px;
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="controlsFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="timeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Local time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="timeZoneHEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="hLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Local time zone (hh)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="timeZoneMEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="mLabel">
        <property name="text">
         <string>Local time zone (mm)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="timeButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>dateTimeChanged()</slot>
 </slots>
</ui>
